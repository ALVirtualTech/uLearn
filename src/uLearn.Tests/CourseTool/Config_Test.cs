using System.IO;
using ApprovalTests;
using NUnit.Framework;

namespace uLearn.CourseTool
{
	[TestFixture]
	public class Config_Test
	{
		[Test]
		public void ConfigTemplate_IsOk()
		{
			var config = new FileInfo("templates\\config.xml").DeserializeXml<Config>();
			config.IgnoredSlides = new[] { "1", "2" };
			Approvals.VerifyXml(config.XmlSerialize());
		}
	}
}