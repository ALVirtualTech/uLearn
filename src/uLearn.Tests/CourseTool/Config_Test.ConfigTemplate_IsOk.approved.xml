<Config>
	<Hostname>openedu.urfu.ru</Hostname>
	<Port>8081</Port>
	<Organization>UrFU</Organization>
	<CourseNumber>IMCS.b.CS-0093</CourseNumber>
	<CourseRun>2015</CourseRun>
	<ExerciseUrl>https://ulearn.azurewebsites.net/Course/{0}/LtiSlide/{1}</ExerciseUrl>
	<SolutionsUrl>https://ulearn.azurewebsites.net/Course/{0}/AcceptedAlert/{1}</SolutionsUrl>
	<LtiId>edx</LtiId>
	<ULearnCourseId>BasicProgramming</ULearnCourseId>
	<Video>video.json</Video>
	<IgnoredUlearnSlides>
		<SlideId>1</SlideId>
		<SlideId>2</SlideId>
	</IgnoredUlearnSlides>
</Config>